modules/sdk-coin-polygon/src/polygon.ts

```diff
@@ -12,13 +12,53 @@ import {
   BuildTransactionParams,
   SignFinalOptions,
 } from '@bitgo/sdk-coin-eth';
-import { BaseCoin, BitGoBase, TransactionExplanation, FullySignedTransaction } from '@bitgo/sdk-core';
+import {
+  BaseCoin,
+  BitGoBase,
+  TransactionExplanation,
+  FullySignedTransaction,
+  SignTransactionOptions,
+  TransactionPrebuild as BaseTransactionPrebuild,
+  HalfSignedAccountTransaction,
+} from '@bitgo/sdk-core';
 import { BaseCoin as StaticsBaseCoin, coins } from '@bitgo/statics';
 import BigNumber from 'bignumber.js';
 import { KeyPair, TransactionBuilder } from './lib';
 import _ from 'lodash';
 import type * as EthTxLib from '@ethereumjs/tx';
 
+interface EIP1559 {
+  maxPriorityFeePerGas: number;
+  maxFeePerGas: number;
+}
+
+interface TransactionPrebuild extends BaseTransactionPrebuild {
+  feeInfo?: EthTransactionFee;
+  source?: string;
+  dataToSign?: string;
+  nextContractSequenceId?: string | number;
+  expireTime?: number;
+  hopTransaction?: string;
+  eip1559?: EIP1559;
+}
+
+interface PolygonSignTransactionOptions extends SignTransactionOptions {
+  txPrebuild: TransactionPrebuild;
+  prv: string;
+  custodianTransactionId?: string;
+}
+
+interface HalfSignedEthLikeTransaction extends HalfSignedAccountTransaction {
```

Comment:
`HalfSignedEthLikeTransaction` as well. Making sure on these since `sdk-coin-polygon` has `abstract-eth` as a dependency with exposed interfaces that may align better (if desired) from there.

Reply:
And Polygon already extends Eth so it matches up from the start, generally.

Reply:
Have updated to use HalfSignedEthLikeTransaction from abstract-eth